### DISCUSSION TITLE
Wrong pypdf version in the docs

### DISCUSSION THREAD

**Discussion by xilopaint** [py-pdf/pypdf#2017] [Q&A] [answered]
When I build the docs to create a [Dash](https://kapeli.com/dash) docset it shows version 3.8.0 rather than the latest version. Why that?

<img width="1440" alt="Screenshot 2023-07-26 at 01 32 46" src="https://github.com/py-pdf/pypdf/assets/11052366/a30d00ec-76d3-49fb-ad47-4a149c46878c">

**Comment by pubpub-zz** [comment]
Can you provide the URL ?

**Reply by xilopaint** [reply]
Do you mean the docset file? You need a macOS machine with Dash installed to open it.

**Comment by pubpub-zz** [comment]
you've written
> When I **build** the docs to create
Can you provide the procedure  you are using ?

**Reply by xilopaint** [reply]
I've built the docs with `make html` and created the docset using [doc2dash](https://github.com/hynek/doc2dash) by running `doc2dash -A _build/html`.

**Reply by pubpub-zz** [reply]
I've run `make html` and the generated pages show 3.13 as expected. maybe you should empty `_build/html` ?

**Reply by xilopaint** [reply]
I've removed the repo, cloned it again and run `make html`. The docset generated by `doc2dash` still shows 3.8.0. Could you suggest something in the docs that has been introduced after 3.8.0 so I can be sure the docs are really new?

**Reply by pubpub-zz** [reply, marked as answer]
when you look at conf.py, you will find that the version is directly pypdf (aliased as py_pkg).__version__
you should run `python -c "import pypdf;print(pypdf.__version__)"` in the same environment you are running make html to confirm you have the good version reported

**Reply by xilopaint** [reply]
Sorry! Now I realize `make html` picks the pypdf installation from my pipenv environment where I had the outdated version installed.

**Reply by xilopaint** [reply]
Btw, I got this errors when building the docs, but despite them the docset was properly generated:

```
reading sources... [100%] user/suppress-warnings
/Users/xxx/.local/share/virtualenvs/pypdf-LoTlALo1/lib/python3.11/site-packages/pypdf/_page.py:docstring of pypdf._page.PageObject.images:13: ERROR: Unexpected indentation.
/Users/xxx/.local/share/virtualenvs/pypdf-LoTlALo1/lib/python3.11/site-packages/pypdf/_page.py:docstring of pypdf._page.PageObject.images:14: WARNING: Block quote ends without a blank line; unexpected unindent.
/Users/xxx/.local/share/virtualenvs/pypdf-LoTlALo1/lib/python3.11/site-packages/pypdf/_writer.py:docstring of pypdf._writer.PdfWriter.set_page_label:14: ERROR: Unexpected indentation.
/Users/xxx/.local/share/virtualenvs/pypdf-LoTlALo1/lib/python3.11/site-packages/pypdf/_writer.py:docstring of pypdf._writer.PdfWriter.set_page_label:15: WARNING: Block quote ends without a blank line; unexpected unindent
```

**Reply by pubpub-zz** [reply]
it looks like some errors within the example. It would be great if you could propose a PR to clean them up

**Reply by xilopaint** [reply]
To fix this we would get some line-too-long Pylint issues. Is this acceptable?

**Reply by pubpub-zz** [reply]
@MartinThoma do you have any recommendations ?

**Reply by MartinThoma** [reply]
It depends. Typically when lines are too long, you would also get issues with black / ruff. And it might look bad in the generated docs.

**Reply by xilopaint** [reply]
Well, we would have one line with 104 chars and two lines with 112 chars. If it's ok, I can submit the PR.
